R/sdk.R

```diff
@@ -0,0 +1,99 @@
+#' Install the Google Cloud SDK
+#'
+#' Installs the Google Cloud SDK which enables CloudML operations.
+#'
+#' @export
+sdk_install <- function() {
+  if (dir.exists(sdk_dir())) {
+    message("SDK already installed.")
+    return(invisible(NULL))
+  }
+
+  if (Sys.info()["sysname"] != "Darwin")
+    stop("Currently, only OS X installation is supported.")
+
+  cloudsdk_url <- "https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/"
+  cloudsdk_tar <- "google-cloud-sdk-178.0.0-darwin-x86_64.tar.gz"
```

Comment:
Should we have some strategy for discovering what the actual current version of the SDK is?

Reply:
Sure! There is more work here to support other OS' etc, but good step in the right direction.